problem while running make sky130hd_temp

finish report_design_area
--------------------------------------------------------------------------
Design area 2551 u^2 19% utilization.

[ERROR GUI-0013] Unable to find visible display control at Pin Markers.
[ERROR GUI-0070] Error: save_images.tcl, 23 GUI-0013
Error: final_report.tcl, 63 GUI-0070
Command exited with non-zero status 1
Elapsed time: 0:00.67[h:]min:sec. CPU time: user 0.54 sys 0.11 (97%). Peak memory: 226936KB.
make[1]: *** [Makefile:568: /home/embedded/Documents/OpenFASOC/openfasoc/generators/temp-sense-gen/flow/logs/sky130hd/tempsense//6_report.log] Error 1
make[1]: Leaving directory '/home/embedded/Documents/OpenFASOC/openfasoc/generators/temp-sense-gen/flow'
[Error] Place and Route failed. Refer to the log file
make: *** [Makefile:48: sky130hd_temp] Error 1




question : 
1. Whats the use of Xyce and trilinos ?
2. How I can solve above problem ?

--- a/openfasoc/generators/temp-sense-gen/flow/scripts/save_images.tcl
+++ b/openfasoc/generators/temp-sense-gen/flow/scripts/save_images.tcl
@@ -20,7 +20,7 @@ gui::set_display_controls "Instances/StdCells/*" visible true
 gui::set_display_controls "Instances/Macro" visible true
 gui::set_display_controls "Instances/Pads/*" visible true
 gui::set_display_controls "Instances/Physical/*" visible true
-gui::set_display_controls "Pin Markers" visible true
+gui::set_display_controls "Pins" visible true
 gui::set_display_controls "Misc/Instances/names" visible true
 gui::set_display_controls "Misc/Scale bar" visible true
 gui::set_display_controls "Misc/Highlight selected" visible true

